utility/src/main/java/org/oppia/util/parser/UrlImageParser.kt

```diff
@@ -60,7 +69,40 @@ class UrlImageParser private constructor(
           } else {
             0
           }
-          val rect = Rect(initialDrawableMargin, 0, drawableWidth + initialDrawableMargin, drawableHeight)
+          val rect =
+            Rect(initialDrawableMargin, 0, drawableWidth + initialDrawableMargin, drawableHeight)
+          drawable.bounds = rect
+          urlDrawable.bounds = rect
+          urlDrawable.drawable = drawable
+          htmlContentTextView.text = htmlContentTextView.text
+          htmlContentTextView.invalidate()
+        }
+      }
+    }
+  }
+
+  private inner class SvgBitmapTarget(private val urlDrawable: UrlDrawable) :
+    CustomTarget<PictureDrawable>() {
+    override fun onLoadCleared(placeholder: Drawable?) {
+      // No resources to clear.
+    }
+
+    override fun onResourceReady(
+      resource: PictureDrawable,
+      transition: Transition<in PictureDrawable>?
+    ) {
+      val drawable = PictureDrawable(resource.picture)
```

Comment:
Why create a copy of the ``PictureDrawable`` vs using the one being passed in? Is it because we're changing the drawable? If so, I suggest adding a brief comment explaining that the duplication is intentional & why since at a glance this could look like an accident that, when changed, might introduce bugs.

Reply:
No we can directly use drawable being passed. It was accidentally written.